### PR TITLE
EMBCESSMOD-1224 - Fixes issue when submitting a new evacuation file for BCSC registered user.

### PR DESCRIPTION
**Pull request by Christopher-Tihor**
Adds Household Member DTO and allows editing of Household Member lists.

### CODE REVIEW

File: evacuees/src/API/EMBC.Registrants.API/EvacuationsModule/EvacuationAutoMapperProfile.cs [hunk 1]
```diff
@@ -45,7 +47,9 @@ public EvacuationAutoMapperProfile()
                 .ForPath(d => d.era_postalcode, opts => opts.MapFrom(s => s.EvacuatedFromAddress.PostalCode))
                 .ForPath(d => d.era_country, opts => opts.MapFrom(s => s.EvacuatedFromAddress.Country.Name))
                 .ForPath(d => d.era_province, opts => opts.MapFrom(s => s.EvacuatedFromAddress.StateProvince.Name))
-                .ForPath(d => d.era_Jurisdiction, opts => opts.MapFrom(s => s.EvacuatedFromAddress.Jurisdiction));
+                .ForPath(d => d.era_Jurisdiction, opts => opts.MapFrom(s => s.EvacuatedFromAddress.Jurisdiction))
+                .ForPath(d => d.era_Jurisdiction.era_RelatedProvinceState.era_code, opts => opts.MapFrom(s => s.EvacuatedFromAddress.Jurisdiction))
+                .ForPath(d => d.era_Jurisdiction.era_RelatedProvinceState.era_RelatedCountry.era_countrycode, opts => opts.MapFrom(s => s.EvacuatedFromAddress.Jurisdiction.CountryCode));
```
Comment (ytqsl) [inline evacuees/src/API/EMBC.Registrants.API/EvacuationsModule/EvacuationAutoMapperProfile.cs:53]: This is probably not going to work, v9.0 doesn't support multi level traversal AFAIK

File: evacuees/src/API/EMBC.Registrants.API/EvacuationsModule/EvacuationAutoMapperProfile.cs [hunk 2]
```diff
@@ -91,9 +96,16 @@ public EvacuationAutoMapperProfile()
                 .ForMember(d => d.HaveSpecialDiet, opts => opts.MapFrom(s => false))
                 .ForMember(d => d.SpecialDietDetails, opts => opts.MapFrom(s => new string(string.Empty)))
                 .ForMember(d => d.HasPetsFood, opts => opts.MapFrom(s => false))
-                .ForMember(d => d.FamilyMembers, opts => opts.MapFrom(s => new List<PersonDetails>()))
+                .ForMember(d => d.HouseholdMembers, opts => opts.MapFrom(s => new List<HouseholdMember>()))
```
Comment (ytqsl) [inline evacuees/src/API/EMBC.Registrants.API/EvacuationsModule/EvacuationAutoMapperProfile.cs:99]: I don't think this member mapping is needed

**Review by ytqsl — CHANGES_REQUESTED** [review]
Few comments

File: evacuees/src/API/EMBC.Registrants.API/EvacuationsModule/EvacuationAutoMapperProfile.cs (hunk 2 above)
Comment (Christopher-Tihor) [reply evacuees/src/API/EMBC.Registrants.API/EvacuationsModule/EvacuationAutoMapperProfile.cs:99]: I added it as the validation test was complaining about its absence.

File: evacuees/src/API/EMBC.Registrants.API/EvacuationsModule/EvacuationAutoMapperProfile.cs (hunk 1 above)
Comment (Christopher-Tihor) [reply evacuees/src/API/EMBC.Registrants.API/EvacuationsModule/EvacuationAutoMapperProfile.cs:53]: Fair enough. I'll get rid of it.

Comment (Christopher-Tihor) [reply evacuees/src/API/EMBC.Registrants.API/EvacuationsModule/EvacuationAutoMapperProfile.cs]: Done!

**Review by ytqsl — APPROVED** [review]
Looks good

**Pull request merged**